frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -616,7 +620,7 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
     {_.isEmpty(props.crdDescs) ? (
       <span className="text-muted">No Kubernetes APIs are being provided by this Operator.</span>
     ) : (
-      props.crdDescs.map((desc) => (
+      props.crdDescs.filter((crd) => !isInternal(crd)).map((desc) => (
```

Comment:
Actually this function is expecting csv object passed with crd name.
In order to reuse  need to pass a custom object with csv and crdDesc